BookingProject/BookingWebAPI/BookingWebApi/Program.cs

```diff
@@ -137,6 +143,28 @@
 });
 builder.Services.AddSingleton<IRedisCacheService, RedisCacheService>();
 
+
+var logger = new LoggerConfiguration()
+    .ReadFrom.Configuration(builder.Configuration)
```

Comment:
?? why code duplication. Just create a separate library, expose extension method like `AddMetrics()` and this way you don't link all libs to new services, but link to a single library